src/gatsby-plugin-theme-ui/components.js

```diff
@@ -23,6 +23,21 @@ import Switch from '../components/organisms/SwitchableMode/Switch'
 import { ReactComponent as GitHubIcon } from '../media/icons/github.svg'
 import { ReactComponent as GitLabIcon } from '../media/icons/gitlab.svg'
 
+const getTextFromPreChildren = preChildren => {
```

Comment:
Looks like we do get some spaces in the new lines, but I think it's still valid YAML 🤔 

<img width="473" alt="image" src="https://user-images.githubusercontent.com/43496356/163845287-0d79e5b4-e3a9-43b4-809c-9eab22eb1c2a.png">